packages/ui/src/components/va-affix/VaAffix.vue

```diff
@@ -27,16 +28,18 @@ import {
   Context,
 } from './VaAffix-utils'
 
+type Target = HTMLElement | Window
+
 export default defineComponent({
   name: 'VaAffix',
   emits: ['change'],
   props: {
     offsetTop: { type: Number as PropType<number>, default: undefined },
     offsetBottom: { type: Number as PropType<number>, default: undefined },
-    target: { type: Function as PropType<() => HTMLElement | Window>, default: () => () => window },
+    target: { type: [Object, Function] as PropType<Target | (() => Target)>, default: getWindow },
   },
   setup (props, { emit }) {
-    const getTargetElement = () => props.target()
+    const getTargetElement = () => (typeof props.target === 'function' ? props.target() : props.target)
```

Comment:
Can this be computed?

Reply:
Ye. But I think it should be another PR. I just fixed it so it work with new withTransportConfig.